packages/core/src/hooks/overlays/useLegacyOverlayStack.ts

```diff
@@ -62,7 +62,7 @@ export function useLegacyOverlayStack(): UseOverlayStackReturnValue {
         legacyGlobalOverlayStackStore.getSnapshot,
     );
 
-    const getLastOpened = React.useCallback(() => stack.at(-1), [stack]);
+    const getLastOpened = React.useCallback(() => stack.at(stack.length - 1), [stack]);
```

Comment:
I think we need to remove `.at` entirely to fix the browser compatibility concern?

```suggestion
    const getLastOpened = React.useCallback(() => stack[stack.length - 1], [stack]);
```

Reply:
I think you're right! Thanks for the feedback.

Reply:
Np! Thanks for the fix.